client/galaxy/scripts/utils/uploadbox.js

```diff
@@ -300,4 +300,3 @@
         };
     }
 })(jQuery);
-
```

Comment:
No, sorry. My text editor automatically removes all trailing spaces and empty lines.